### DISCUSSION TITLE
请问istore-reinstall是重新安装istore还是恢复出厂设置？

### DISCUSSION THREAD

**Discussion by a447948814** [istoreos/istoreos#940] [Q&A] [answered]
![image](https://github.com/istoreos/istoreos/assets/61766946/3a8db966-e516-4e1d-b7d5-6d3b7dd1f482)
如图这个选项是重新安装系统还是重置设置？

**Comment by jjm2473** [comment, marked as answer]
只是重装istore商店而已，你们怎么老是把istore跟istoreos搞混呢？

**Reply by a447948814** [reply]
哈哈 好嘞 ，感谢修复系统软件功能！解决了很多问题。比如 无法生成备份。无法更新软件包。docker选项消失等等。
